config/initializers/ezid_client.rb

```diff
@@ -0,0 +1,6 @@
+Ezid::Client.configure do |config|
+  config.default_shoulder = "ark:/99999/fk4"
+  config.user = "apitest"
+  config.password = "apitest"
```

Comment:
I think this needs to be managed by the Ansible playbook as it holds different configurations for prod and test/dev environments.